docs/eslint.config.mjs

```diff
@@ -0,0 +1,273 @@
+import nextPlugin from '@next/eslint-plugin-next';
+import { defineConfig, globalIgnores } from 'eslint/config';
+import universeNodeConfig from 'eslint-config-universe/flat/node.js';
+import universeTypescriptAnalysisConfig from 'eslint-config-universe/flat/shared/typescript-analysis.js';
+import universeWebConfig from 'eslint-config-universe/flat/web.js';
+import lodash from 'eslint-plugin-lodash';
+import * as mdx from 'eslint-plugin-mdx';
+import tailwind from 'eslint-plugin-tailwindcss';
+import testingLibrary from 'eslint-plugin-testing-library';
+import unicorn from 'eslint-plugin-unicorn';
+
+const TAILWIND_DEFAULTS = {
+  callees: ['mergeClasses'],
+  classRegex: '^(confirmation|container|icon)?(c|C)lass(Name)?$',
+};
+
+const CORE_RULES = {
+  'prettier/prettier': 'error',
+  'no-void': ['warn', { allowAsStatement: true }],
+  'no-return-await': 'off',
+  // 'import/order': [
+  //   'error',
+  //   {
+  //     groups: [['external', 'builtin'], 'internal', ['parent', 'sibling']],
+  //     'newlines-between': 'always',
+  //     alphabetize: {
+  //       order: 'asc',
+  //     },
+  //     pathGroups: [
+  //       {
+  //         pattern: '~/**',
+  //         group: 'internal',
+  //       },
+  //     ],
+  //   },
+  // ],
+  curly: 'warn',
+  eqeqeq: ['error', 'always', { null: 'ignore' }],
+  // 'import/no-cycle': ['error', { maxDepth: '∞' }],
```

Comment:
Disabled to prevent making changes in the codebase, but here's an example of it - looks valid to me!

<img width="1245" alt="Screenshot 2025-04-16 at 14 23 44" src="https://github.com/user-attachments/assets/433fe2d2-4285-46b6-b192-65278f89196b" />

Reply:
No it was there before, but I upgraded the import plugin and I think it's become better at spotting these.
As discussed we'll address these in a follow-up PR and will leave it as commented out for now.